lectures/06_lecture/deployment.ipynb

```diff
@@ -0,0 +1,206 @@
+{
```

Comment:
This notebook looks great :+1: 
Just an idea in case you have some time left in the end of the lecture - you can mention docker-compose for running multi-container Docker applications where you can separate logical units like API, ML model, database, ... into separate environments.